pages/windows/finger.md

```diff
@@ -0,0 +1,20 @@
+# finger
+
+> Return information about one or more users on a specified system.
+> The remote system must be running the Finger service.
+
+- Display information about a specific user:
+
+`finger {{user}}@{{host}}`
+
+- Display information about all users on the specified host:
+
+`finger @{{host}}`
+
+- Display information in a longer format:
+
+`finger {{user}}@{{host}} -l`
```

Comment:
You use a `-l` here, but a `/?` further down. Typo?

Reply:
Not a typo, Microsoft is just inconsistent. You can use `--help` (instead of `/?`) but I wasn't sure if that's available in older versions of the command.

Reply:
Ah, I see! Good ol' microsoft :P